### ISSUE TITLE
I can't get the same Chamfer Distance Results. Should I set the weight coefficient of chamfer distance loss to 1 to fine tune it? If so, how many epochs should I train?

### ISSUE THREAD

**Issue by fyyyym** [PRBonn/point-cloud-prediction#15]
I trained the model on a single 4090D GPU for 50 epochs. And I set the weight coefficient of  chamfer distance loss to 0. But I can't get the same Chamfer Distance Results on the KITTI as inferred in the paper.(the value is way higher). 
Should I set the weight coefficient of chamfer distance loss to 1 to fine tune it? If so, how many epochs should I train?

**Comment by benemer** [comment]
Which value are you getting, and to which result of the paper are you referring?

For example, we obtained the results in Table 1 by training for 50 epochs with only range and mask loss and then fine-tuning with a Chamfer distance loss weight of 1.0 for ten epochs as specified in the caption.

In general, worse Chamfer distance results are expected if the weight is set to zero, see Table 2.

**Comment by fyyyym** [comment]
I see.
Thank you very much!
